<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsCSS</class>
 <widget class="QWidget" name="settingsCSS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>41</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="backgroundLabel">
     <property name="text">
      <string>CSS theme:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="kcfg_css_theme"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
